hyperSpec/_pkgdown.yml

```diff
@@ -9,7 +9,10 @@ template:
 # -------------------------------------------------------------------------
 
 development:
-  mode: auto
+  mode: devel
+  destination: vig
+  version_label: warning
+  version_tooltip: "Website to preview the output of vignettes"
```

Comment:
Before merging, these lines should be reverted (I'll do it myself).
They were needed to build **pkgdown** website and to deploy to the `vig` folder on the website.